perses/app/relative_point_mutation_setup.py

```diff
@@ -224,6 +230,10 @@ def __init__(self,
                                                                    validate_energy_bookkeeping=validate_bool)
             logp_reverse = geometry_engine.logp_reverse(topology_proposal, new_positions, pos, beta,
                                                         validate_energy_bookkeeping=validate_bool)
+            
+            if old_system is not None and new_system is not None:
```

Comment:
What would happen if only one was passed in?

Reply:
maybe i should throw an error if only one is passed in. or i can have the `PointMutationExecutor` generate the system that wasn't passed in. @dominicrufa  thoughts?

Reply:
i don't necessarily think that this should stay. its is only there for debugging purposes